crates/oxc_linter/src/rules/import/no_amd.rs

```diff
@@ -0,0 +1,93 @@
+use oxc_ast::ast::Expression;
+use oxc_diagnostics::{
+  miette::{self, Diagnostic},
+  thiserror::Error,
+};
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{Atom, ModuleKind, Span};
+use oxc_ast::AstKind;
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+#[derive(Debug, Error, Diagnostic)]
+#[error("eslint-plugin-import(no-amd): Do not use AMD `require` and `define` calls.")]
+#[diagnostic(severity(warning), help("Consider change {1} to EMScript import or export"))]
```

Comment:
```suggestion
#[diagnostic(severity(warning), help("Consider changing {1} to ECMAScript import or export"))]

Reply:
Thanks, I updated the codes like you suggested.

Reply:
Updated.